<?xml version="1.0" encoding="UTF-8"?>
<launch>
  <arg name="input/pointcloud" default="/kitti/velo/pointcloud"/>
  <arg name="output/pointcloud" default="/output/pointcloud"/>
  <arg name="config_file" default="$(find-pkg-share patchworkpp)/config/patchworkpp.params.yaml"/>
  <arg name="debug" default="true"/>

  <node pkg="patchworkpp" exec="patchwork_pp" name="patchwork_pp" output="screen">
    <remap from="input/pointcloud" to="$(var input/pointcloud)"/>
    <remap from="output/pointcloud" to="$(var output/pointcloud)"/>
    <param from="$(var config_file)"/>
    <param name="debug" value="$(var debug)"/>
  </node>
</launch>
